channel.go

```diff
@@ -0,0 +1,340 @@
+package cli
+
+import (
+	"encoding/json"
+	"time"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/cheynewallace/tabby"
+	"github.com/urfave/cli/v2"
+)
+
+func NewChannelCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "channel",
+		Usage:       "Manage channels.",
+		Description: "Manage channels.",
+		Subcommands: []*cli.Command{
+			getChannelCmd(config),
+			newListChannelsCmd(config),
+			newCreateChannelCmd(config),
+			newDeleteChannelCmd(config),
+			newUpdateChannelCmd(config),
+		},
+	}
+}
+
+func getChannelCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "get",
+		Usage:       "Get a channel by channel type and channel name.",
+		Description: "Get a channel by channel type and channel name.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "channel-type",
```

Comment:
`type` is enough, it's explicit we are dealing with channels here
```suggestion
				Name:     "type",
```

Reply:
👍